src/plugins/profiling/server/routes/flamegraph.ts

```diff
@@ -7,7 +7,8 @@
  */
 
 function getExeFileName(exe: any, type: number) {
-  if (exe.FileName === undefined) {
+  if (exe === undefined || exe.FileName === undefined) {
```

Comment:
Minor suggestion: you can use optional chaining to simplify the check

```suggestion
  if (exe?.FileName === undefined) {
```